src/SFA.DAS.EmployerIncentives/Application/Services/EmployerIncentivesService.cs

```diff
@@ -96,6 +104,13 @@ public async Task<IncentiveApplicationDto> GetApplication(long accountId, Guid a
             return response;
         }
 
+        public async Task<long> GetApplicationLegalEntity(long accountId, Guid applicationId)
+        {
+            var response = await _client.Get<long>(new GetApplicationLegalEntityRequest(accountId, applicationId));
+
+            return response;
+        }
+
```

Comment:
would it be better to simply get the LegalEntityId from the Application. As it's being added there. Saves this extra call

Reply:
Two reasons for this endpoint: One is that it seems overkill to pass large amounts of JSON (full application and all the apprenticeships data) back just to retrieve the AccountLegalEntityId. Second is that this endpoint could be used by the EI-40 story, to get the AccountLegalEntityID if it's not supplied in the URL

Reply:
I can see you're logic here you only need one field. Should the method be called 'GetApplicationAccountLegalEntityId' as that is what it's doing. Or are we planning on adding the AccountLegalEntityId and the LegalEntityId to this call in the future

Reply:
I've renamed in this branch to hopefully avoid confusion:
https://github.com/SkillsFundingAgency/das-apim-endpoints/pull/85/commits/cccb522ece4707229fb5f0efe26b9937780e5906